s3-source-connector/src/test/java/io/aiven/kafka/connect/s3/source/utils/SourceRecordIteratorTest.java

```diff
@@ -17,6 +17,7 @@
 package io.aiven.kafka.connect.s3.source.utils;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.anyLong;
```

Comment:
*Really* not important -- but consider using `org.mockito.Mockito` for consistency!

Reply:
Thank you. Updated. Not sure how it ended up there.